docs/site/contest_formats.md

```diff
@@ -28,16 +37,28 @@ The score is equal to the sum of scores of the last submissions to each problem.
 By default, ties are not broken, however, setting `cumtime` to `true` will sum the times of the last submission to each problem, and use that to tiebreak.
 
 The `first_ac_bonus` field, as suggested by the name, will add the specified number of points to a problem's score if it is solved on the first attempt, excluding compile errors and internal errors.
-This field defaults to `10`.
+This field defaults to 10.
 
 The `time_bonus` field awards a bonus for solving problems faster.
-The field is in minutes, and for each such interval of time before the contest ends, any submission with a non-zero score will have a bonus added. This field defaults to `5`.  
+The field is in minutes, and for each such interval of time before the contest ends, any submission with a non-zero score will have a bonus added. This field defaults to 5.  
 Note that specifying `0` will disable this bonus.  
 For example, say a submission with a score of `50/100` is submitted 23 minutes before the contest ends. $\left\lfloor\frac{23}{5}\right\rfloor = 4$, so 4 bonus points are awarded, giving a total score of $50 + 4 = 54$ for that problem.
 
 ## AtCoder Contest Format
 
-The `AtCoder` contest format is based on the contest format used by AtCoder. As such, your score equals the sum of your highest-scoring submission on each problem, and ties are broken based on the time of your last score-changing submission plus your penalty.
+The `AtCoder` contest format is based on the contest format used by AtCoder.
+
+The score is equal to the sum of the highest-scoring submission on each problem, and ties are broken based on the time of the last score-changing submission plus the penalty.
 
 The penalty is specified by the `penalty` field, and defaults to 5 minutes.
 The penalty is equal to the total number of incorrect submissions prior to the highest scoring submission on each solved problem, multiplied by the specified value, in minutes, and is added to the cumulative time.
+
+## ICPC Contest Format
+
+The `ICPC` contest format is based on the contest format used by the ICPC.
+
+The score is equal to the number of problems solved, and ties are broken firstly based on the sum of the elapsed time that a correct submission was submitted to each problem plus the penalty and secondly based on the time of the last score-changing submission. 
+
+The penalty is specified by the `penalty` field, and defaults to 20 minutes.
+The penalty is equal to the number of incorrect submissions prior to the first correct submission, multiplied by the specified value, in minutes, and is added to the cumulative time. 
+Note that there is no time penalty for a problem that is not solved.

```

Comment:
I believe you get penalty as long as you have a non-zero score (solved implies AC)

Reply:
There are no partials.

Reply:
Incorrect: this format can be used on contests that have partials enabled.